blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -3,19 +3,23 @@ import Immutable from 'immutable';
 since we would return a function, */
 /* eslint new-cap: [0] */
 const { Record } = Immutable;
-export const PipelineRecord = Record({
+export class PipelineRecord extends Record({
```

Comment:
This makes no sense to me, why would you extend a reference which is then thrown away?